platform-ui-skin/pom.xml

```diff
@@ -37,6 +37,7 @@
             <plugin>
                 <groupId>org.lesscss</groupId>
                 <artifactId>lesscss-maven-plugin</artifactId>
+                <version>1.7.0.1.1</version>

```

Comment:
Thanks for the update.
My point about the lesscss maven plugin is that you should not update its version in this PR, and it should compile as is. Can you try ?